app/views/components/link.scala.html

```diff
@@ -14,6 +14,14 @@
  * limitations under the License.
  *@
 
-@(href : String, id : String = "", key : String, openInNewWindow: Boolean = false)(implicit messages: Messages)
+@import viewmodels.Link
 
-<a id="@id" href="@href" @if(openInNewWindow) {target="_blank"}>@messages(key)</a>
\ No newline at end of file
+@(link: Link)(implicit messages: Messages)
```

Comment:
@jackheslop96 is there a better naming convention for this? I noticed you've moved the original implementation into a new registration-progress-link.